### ISSUE TITLE
Create template values files for a basehub and a daskhub

### ISSUE THREAD

**Issue by sgibson91** [2i2c-org/infrastructure#1798]
It would be useful now to have some `basehub-common-template.values.yaml` and `daskhub-common-template.values.yaml` files that have the bare minimum required to copy from, since it is becoming more difficult to determine what is and what isn't customised just by inspecting _other_ hub's values files.

For example I believe when deploying the NASA cryo hubs, I faced some issues because I was copying Openscapes config that had particular customisations that it was difficult to know about until it caused issues.

* https://github.com/2i2c-org/infrastructure/issues/1768

**Comment by GeorgianaElena** [comment]
+100000000000000 on this

At some point I was actually thinking that we could have more charts than the basehub and daskhub charts. Something like: `edu-hub`, `pangeo-hub`, `database-hub`, and whatever other sub-types we identify. But that might have been too much. But the reasoning behind is the same.

**Comment by sgibson91** [comment]
> we could have more charts than the basehub and daskhub charts. Something like: `edu-hub`, `pangeo-hub`, `database-hub`, and whatever other sub-types we identify.

I think where we would like to head is that all of our different features are simply `enabled: true` flags to basehub(/daskhub) and template values files for `<hub-name>.values.yaml` or `common.values.yaml` helps us more easily setup things like: NFS IP addresses, annotations, etc.

**Comment by sgibson91** [comment]
@GeorgianaElena I know you did a lot of work on creating some templates and we now have a `deployer generate-<cloud>-cluster` command. What I'd really like it a `deployer generate-<cloud>-hub` command for scenarios like adding a new hub to an existing cluster. I'm not sure if that work is being tracked elsewhere or if it would be appropriate to reframe this issue to that end?

**Comment by GeorgianaElena** [comment]
> What I'd really like is a deployer generate-<cloud>-hub command for scenarios like adding a new hub to an existing cluster. I'm not sure if that work is being tracked elsewhere or if it would be appropriate to reframe this issue to that end?

@sgibson91, yes me too! It was tracked in the old goal https://github.com/2i2c-org/infrastructure/issues/1917 issue, 
<img width="865" alt="Screenshot 2023-10-18 at 17 37 21" src="https://github.com/2i2c-org/infrastructure/assets/7579677/35749450-589b-42a8-9559-3a57ba63501f">

But since that is closed, I believe it's a great idea to have it described here. Thank you!
